examples/knative/powercli/kn-pcli-nsx-tag-sync/handler.ps1

```diff
@@ -181,18 +186,22 @@ Function Process-Handler {
 
    # POST to NSX
    try {
-      $response = ""
+      if (${env:FUNCTION_DEBUG} -eq "true") {
+         Write-Host "$(Get-Date) - DEBUG: NSX Body=$($nsxBody)"
+      }
+
+      $encodednsxbody = [System.Text.Encoding]::UTF8.GetBytes($nsxbody);
```

Comment:
question: should this actually be in the `Write-Host` line above so that you see the converted body instead of "before"? (eases troubleshooting)

Reply:
I intentionally had the 'before' because I wanted to see if the umlaut character was even making it to the variable. Maybe I should display both before and after.

Reply:
not sure if it's needed tbh, since it's just an intermediate representation. but fine with me if you want to log both

Reply:
Actually this isn't going to work. The encoding process turns the payload into a byte array.  To display it, I'd have to decode it. But I already show the decoded payload by printing $nsxBody

DEBUG: Encoded NSX Body=123 10 32 32 34 116 97 103 115 34 58 32 91 10 32 32 32 32 123 10 32 32 32 32 32 32 34 115 99 111 112 101 34 58 32 34 84 101 115 116 84 97 103 115 34 44 10 32 32 32 32 32 32 34 116 97 103 34 58

Reply:
cool then!